### DISCUSSION TITLE
$this->reset('prop') disappears if set to null

### DISCUSSION THREAD

**Discussion by AdamEsterle** [livewire/livewire#6772] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Make a component with a property set to null
`public $selectedUserId = null;`
2. Reset the property
`$this->reset('selectedUserId');`
3. Try to access the property
4. Notice the property fully disappears (does not exist) and is not just set to null (as expected and what happened in Livewire 2.x)

`Livewire\Exceptions\PropertyNotFoundException: Property [$selectedUserId] not found on component:`

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@AdamEsterle this has been fixed in PR #6729 and will be in the next release.
